pandas/core/util/numba_.py

```diff
@@ -11,6 +11,14 @@
 
 from pandas.util.version import Version
 
+try:
+    # Reduce likelihood of segfaulting during parallelism
+    # https://github.com/numba/numba/issues/6839#issuecomment-868352174
```

Comment:
Numba may print a message to `STDERR` to warn users that their code is not compatible with the current threading backend and issue a `SIGABRT`. If the Numba threading backends or runtime segfaults (`SIGSEGV`), it's probably a bug!